### ISSUE TITLE
Add meta info of Label Control in text classification task

### ISSUE THREAD

**Issue by supercoderhawk** [HumanSignal/label-studio#1328]
**Is your feature request related to a problem? Please describe.**

When I annotate text classification task, I want to add some additional text of the annotated label. This additional text is just confirmed by current annotated text, can't be listed beforehand.

**Describe the solution you'd like**

For example 

I want to annotate following text within predefined labels `Chemical Drug`, `Biologics` and `Vaccine`

```
A compound 2-acetoxybenzoic acid can be used to treat Inflammation.

```

This is annotated `Chemical Drug`. But I want to add a tag `aspirin` on  this annotated label. This is not supported currently.

**Describe alternatives you've considered**

I formulate this text classification task as a NER task to annotate it. I use entity type as text type,  and select some words as region in text randomly and add tag what I required as meta info of this region.

**Comment by niklub** [comment]
Hi, @supercoderhawk ! Thanks for bringing it up. Do you think this option is something you're looking for?
![image](https://user-images.githubusercontent.com/6087484/130039598-2fd28ac7-7fac-489a-998b-4be860d57fbb.png)

**Comment by supercoderhawk** [comment]
Yes. If you bind this part to Label control, and I will appreciate it.

**Comment by niklub** [comment]
What do you mean exactly from binding it to Label control? do you want to streamline the flow in UI to create meta info right after drawing a span? Since technically, once you create meta as in the example above, it becomes connected to the labeled region.
Speaking of a more streamlined approach, you can alternatively attach `<TextArea>` tag to your label, then it gives you the handy way to create meta info attached to the Label, like in the screen bellow:
![image](https://user-images.githubusercontent.com/6087484/130067569-e84c1b8f-46b0-4536-afe6-563f844c4f55.png)

Please, check the flow with this labeling configuration, and happy to get your feedback on it!
```xml
<View>
  <Labels name="label" toName="text">
    <Label value="Chemical Drug" background="red"/>
    <Label value="Biologics" background="darkorange"/>
    <Label value="Vaccine" background="orange"/>
  </Labels>
  <TextArea name="transcription" toName="text" editable="true" perRegion="true" required="true" maxSubmissions="1" rows="5" placeholder="Meta Info" displayMode="region-list"/>
  <Text name="text" value="$text"/>
</View>
```

**Comment by supercoderhawk** [comment]
Sorry, I mislead you. I mean let Choice control be associated with (or binded with) meta info box, such as you first reply, not Lable control. When I select a Choice box, I can add meta info on it.
